src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/QCStatusSort.java

```diff
@@ -0,0 +1,46 @@
+package ca.on.oicr.gsi.dimsum.service.filtering;
+
+import java.util.Comparator;
+import java.util.List;
+import ca.on.oicr.gsi.cardea.data.Sample;
+
+public class QCStatusSort implements Comparator<Sample> {
```

Comment:
This can be included in the `SampleSort` enum declaration since it's not used elsewhere. See `CaseSort.URGENCY` for an example of what this looks like